src/swarm/node/model/NeoNode.d

```diff
@@ -618,7 +618,7 @@ public abstract class INodeBase : INode, INodeInfo
 
 *******************************************************************************/
 
-public class NodeBase ( ConnHandler : ISwarmConnectionHandler ) : INodeBase
+public abstract class NodeBase ( ConnHandler : ISwarmConnectionHandler ) : INodeBase
```

Comment:
`abstract`

Reply:
This is still abstract (it shouldn't be).

Reply:
Updated.